articles/index.md

```diff
@@ -68,37 +68,37 @@ Develop a [theory of change](/theory) for your activist organisation using our t
 
 <action-smart-large/>
 
-## We work with data & revolutionary tech
+## We are working with data & revolutionary technologies
```

Comment:
Debatable, my own issue was that the "data &" was on a new line alone. IDK how this will even look like once it renders, and my assumption that changing the column width would be too brutal.

Reply:
Changed to "Our research & revolutionary tech", but it's different on every screen anyways, so never really optimal